src/mount/mastercomm.cc

```diff
@@ -80,6 +82,15 @@ typedef struct _acquired_file {
 	struct _acquired_file *next;
 } acquired_file;
 
+#ifdef _WIN32
+// These default values enable setting all requested inodes to "no permission,"
+// thereby preventing interaction with them by all users, allowing the client to
+// operate without interruption.
+constexpr uint8_t defaultFsSendAndReceiveResponse[] = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
```

Comment:
Where is this variable used?

Reply:
After new considerations there was a simplification of the implementation as it is not necessary with the latest approach